tools/installpsh-amazonlinux.sh

```diff
@@ -0,0 +1,221 @@
+            PSUEDONAME=`cat /etc/SuSE-release | tr "\n" ' '| sed s/VERSION.*//`
+            REV=`cat /etc/SuSE-release | grep 'VERSION' | sed s/.*=\ //`
+        elif [ -f /etc/mandrake-release ] ; then
+            DistroBasedOn='mandrake'
+            PSUEDONAME=`cat /etc/mandrake-release | sed s/.*\(// | sed s/\)//`
+            REV=`cat /etc/mandrake-release | sed s/.*release\ // | sed s/\ .*//`
+        elif [ -f /etc/debian_version ] ; then
+            DistroBasedOn='debian'
+            DIST=`cat /etc/lsb-release | grep '^DISTRIB_ID' | awk -F=  '{ print $2 }'`
+            PSUEDONAME=`cat /etc/lsb-release | grep '^DISTRIB_CODENAME' | awk -F=  '{ print $2 }'`
+            REV=`cat /etc/lsb-release | grep '^DISTRIB_RELEASE' | awk -F=  '{ print $2 }'`
+        fi
+        if [ -f /etc/UnitedLinux-release ] ; then
+            DIST="${DIST}[`cat /etc/UnitedLinux-release | tr "\n" ' ' | sed s/VERSION.*//`]"
+        fi
+        OS=`lowercase $OS`
+        DistroBasedOn=`lowercase $DistroBasedOn`
+        readonly OS
+        readonly DIST
+        readonly DistroBasedOn
+        readonly PSUEDONAME
+        readonly REV
+        readonly KERNEL
+        readonly MACH
+    fi
+
+fi
+
+if [ "$DistroBasedOn" != "$thisinstallerdistro" ]; then
+  echo "*** This installer is only for $thisinstallerdistro and you are running $DistroBasedOn, please run \"$gitreporoot\install-powershell.sh\" to see if your distro is supported AND to auto-select the appropriate installer if it is."
+  exit 0
+fi
+
+## Check requirements and prerequisites
+
+#Only do SUDO if we are not root
+SUDO=''
+if (( $EUID != 0 )); then
+    SUDO='sudo'
+fi
+
+#Check that sudo is available
+if [[ "$SUDO" -eq "sudo" ]]; then
+
+    $SUDO -v
+    if [ $? -ne 0 ]; then
+      echo "ERROR: You must either be root or be able to use sudo" >&2
+      exit 5
+    fi
+fi
+
+#Collect any variation details if required for this distro
+
+#END Verify The Installer Choice
+
+echo
+echo "*** Installing prerequisites for PowerShell Core..."
+$SUDO yum install -y \
+        curl \
+        libunwind \
+        libicu \
+        libcurl \
+        openssl \
+        libuuid.x86_64 \
+    && yum clean all
+
+##END Check requirements and prerequisites
+
+echo
+echo "*** Installing PowerShell Core for $DistroBasedOn..."
+release=`curl https://api.github.com/repos/powershell/powershell/releases/latest | sed '/tag_name/!d' | sed s/\"tag_name\"://g | sed s/\"//g | sed s/v//g | sed s/,//g | sed s/\ //g`
+
+#DIRECT DOWNLOAD
+pwshlink=/usr/bin/pwsh
+package=powershell-${release}-linux-x64.tar.gz
+downloadurl=https://github.com/PowerShell/PowerShell/releases/download/v$release/$package
+
+echo "Destination file: $package"
+echo "Source URL: $downloadurl"
+
+curl -L -o "$package" "$downloadurl"
+
+if [[ ! -r "$package" ]]; then
+    echo "ERROR: $package failed to download! Aborting..." >&2
+    exit 1
+fi
+
+echo "Installing PowerShell to /opt/microsoft/powershell/$release in overwrite mode"
```

Comment:
Maybe add this line to `installpsh-suse.sh` too?

Reply:
For this comment and the one below, they are about adding the `ehco message` to the same place in `installpsh-suse.sh`. Making this change won't need additional tests as it's just adding a log message. Could you please do it?

Reply:
This change is made, squashed and pushed - but this PR is now showing as "closed" ?

Reply:
@DarwinJS The timeline of this PR shows that you closed it ... But I see you already opened a new PR.